tests/functions.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+require_once '../Album.php';
+require_once '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class functions extends TestCase
+{
+    public function testIntToBritDate_Given20201004return04102020()
```

Comment:
No need to change, but try and be more generic with test names in the future.

Something more like `testIntToBritDate_correctDate` would do the job